<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportProfileDlg</class>
 <widget class="QDialog" name="exportProfileDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import Export Profiles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Action</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="importRadio">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="exportRadio">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selProfilesBtn">
     <property name="text">
      <string>Select Profiles</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selSpcCollectionsBtn">
     <property name="text">
      <string>Select Species Collections</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selMetaGroupsBtn">
     <property name="text">
      <string>Select Metadata Groups</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
